.github/workflows/cockpit-lib-update.yml

```diff
@@ -24,6 +25,8 @@ jobs:
 
       - name: Clone repository
         uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.INSTALLKER_TOKEN }}
```

Comment:
I think you have a typo here? INSTALLKER?

Also the fact that we can't update is not making this PR blocked. We can nicely generate the PRs just not merge them.

Reply:
That's the bot name, so its token has its name.